### DISCUSSION TITLE
docs/cn/

### DISCUSSION THREAD

**Discussion by giscus** [RavenfieldCommunity/docs-comment#2] [Announcements]
# docs/cn/

Ravenfield社区客制化支持 Raven中文分部(bushi 基于CC-BY-NC-4.0 (https://github.com/RavenfieldCommunity/docs/blob/master/LICENSE)许可授权的Ravenfield社区客制化文档💡 想要帮助我们维护文档？ >>>项目维护帮助 (/cn/CONTRIBUTIN...

https://ravenfieldcommunity.github.io/docs/cn/

**Comment by KeilAwk** [comment]
# TEST
Hello,Ravenfield!
